src/Core/src/Stream/ResponseBodyStream.php

```diff
@@ -4,12 +4,17 @@
 
 namespace AsyncAws\Core\Stream;
 
+use AsyncAws\Core\Exception\LogicException;
 use Symfony\Contracts\HttpClient\ResponseStreamInterface;
 
 /**
  * Stream a HTTP response body.
+ * This class is a BC layer for Http Response that does not supports toStream.
```

Comment:
```suggestion
 * This class is a BC layer for Http Response that does not support toStream().
```

Removed `supports` and make sure `toStream` looks like a function.